pkg/features/kube_features.go

```diff
@@ -526,7 +519,6 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	EphemeralContainers:                 {Default: false, PreRelease: featuregate.Alpha},
 	PodShareProcessNamespace:            {Default: true, PreRelease: featuregate.Beta},
 	PodPriority:                         {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in 1.18
-	TaintNodesByCondition:               {Default: true, PreRelease: featuregate.Beta},
```

Comment:
If we're going to move this feature to GA, we shouldn't be removing the feature gate in the same release.

Reply:
I removed the feature gates because of Klaus's https://github.com/kubernetes/kubernetes/pull/82703#discussion_r324420293 here...

cc/ @k82cn